CHANGELOG.rst

```diff
@@ -167,56 +167,57 @@ v0.7.0 RC2
 **Maintenance**
 
 * Documentation Changes
-  * Credit Rochman for allowing redistribute TENDL file
-  * Fix various typos
-  * automatic deployment of a updated version of the website on tags
-  * automatic creation of a new version of the website (not deployed) for
+   * Credit Rochman for allowing redistribute TENDL file
```

Comment:
This whitespace change may be "right" for RST, but I'm not sure I want to put it in now as it will wreak havoc on all our other PRs.  Not sure if you have a way to produce this CHANGELOG without the whitespace changes?

Reply:
Actually those were intentional because the formatting was messed up. I can submit that as a separate PR, though

Reply:
I actually noticed even more formatting errors after this too -- but it definitely makes more sense to have it separate